stonesoup/detector/beamformers.py

```diff
@@ -0,0 +1,483 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import math
+import random
+import numba
```

Comment:
I'm not sure we want to include `numba` dependency for the one function below. Suggest removing this for now.

Separately I'd be interested if we could use `numba` in certain places however.

Reply:
Good spot @sdhiscocks, I'd mistakenly left that function in from an earlier version which was doing some complex number operations. I've removed both `numba` and the function now.